packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker/create-transform-job.ts

```diff
@@ -59,6 +59,13 @@ export interface SageMakerCreateTransformJobProps extends sfn.TaskStateBaseProps
    */
   readonly modelName: string;
 
+  /**
+   * Configures the timeout and maximum number of retries for processing a transform job invocation.
+   *
+   * @default
+   */
+  readonly modelClientConfig?: ModelClientConfig;
```

Comment:
is it possible to specify this property through state input? (i.e. a string that says `$.field` - if so we might need an enum like class here. if not, you can disregard this comment.